dev/mima

```diff
@@ -41,8 +41,16 @@ else
   JAVA_CMD=java
 fi
 
+
+# Workaround to make mima runs on Java 11+
+JAVA_VER=$(java -version 2>&1 | head -1 | cut -d'"' -f2 | sed '/^1\./s///' | cut -d'.' -f1)
+if [[ JAVA_VER -ge 11 ]]; then
+  EXTRA_JAVA_OPTIONS="--add-opens=java.base/java.util.jar=ALL-UNNAMED"
+fi
+
 $JAVA_CMD \
   -Xmx2g \
+  $EXTRA_JAVA_OPTIONS \
```

Comment:
Thank you for update. Let's simplify like this by removing line 44~50 and add directly.
```
$JAVA_CMD \
  -Xmx2g \
  -XX:+IgnoreUnrecognizedVMOptions --add-opens=java.base/java.util.jar=ALL-UNNAME \
  ...
```